File: public/section05/06.js
/**
 * 학급 회장(해쉬)
 */

function solution(n = 0, s = '') {
  //
  const res = {};

  for (let char of s) {
    if (char in res) {
      res[char] += 1;
    } else {
      res[char] = 1;
    }
  }

  let arr = Object.values(res);
  let max = Math.max(...arr);
  console.log(max);

  return Object.keys(res).find(key => res[key] === max);
}

console.log(solution(15, 'BACBACCACCBDEDE'));

/**
 *
 */

function solution1() {
  //
}

console.log(solution1(15, 'BACBACCACCBDEDE'));

File: public/section01/06.py
"""
"""

def solution(arr):

  sum = 0
  min = 1000

  for num in arr:
    if num % 2 == 1:
      sum += num
      if num < min:
        min = num

  return [sum, min]

print(solution([12, 77, 38, 41, 53, 92, 85]))